lsst/ts/laser/csc.py

```diff
@@ -82,7 +87,8 @@ def __init__(self, address, configuration, frequency=1, initial_state=State.STAN
                 self.fault()
```

Comment:
When a component goes into fault state you need to publish a errorCode event. I think this is a good opportunity to implement that here.